fieldservice_route/models/fsm_order.py

```diff
@@ -1,5 +1,6 @@
 # Copyright (C) 2019 Open Source Integrators
 # Copyright (C) 2019 Serpent consulting Services
+# Copyright 2022 Tecnativa - Víctor Martínez
```

Comment:
@victoralmau Thank you for PR.

Can you please remove your **Copyright** and add in the contributor's fragment?

Reply:
Changes done.

Reply:
If he changes the file, the copyright should be there.